prometheus_all/input.tf

```diff
@@ -114,6 +119,10 @@ variable "postgres_services" {
   description = "List of postrgres services for advanced monitoring and dashboard. [\"space1/postgres1\", \"space2/postgres2\", ...]"
   default     = []
 }
+variable "alertable_postgres_services" {
+  description = "List of postrgres services that will have alerting enabled. [\"space1/postgres1\", \"space2/postgres2\", ...]"
```

Comment:
Typo: postrgres

Reply:
Fixed typo (and the one above that I copied it from)